doc/specs/userLog.md

```diff
@@ -0,0 +1,109 @@
+# User log
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+User log is where notable events or messages are recorded. These can be:
+* User actions.
+* Administrator actions or messages against the user.
+* System events impacting the user.
+
+## Business Rules
+
+* Messages in the user log are subject to visibility permissions through tagging.
+  * Messages with a tag are subject to that tag's viability restrictions.
+* Messages can be filtered by date, time, tag, user and message.
+
+## Use Cases
+
+### 1.0 Viewing the user log
+
+#### 1.1 As a user
+
+**Primary Actor:** A user.
+
+**Brief:** The user log is visible to the user.
+
+**Preconditions:**
+* The user log contains messages with varying tags.
+
+**Trigger:** User wants to see their own user log.
+
+**Postconditions:**
+* No messages outside of the user's permissions were shown.
+
+**Basic Flow:**
+
+1. The user navigates to their user log page.
+2. The user log messages are displayed.
+
+#### 1.2 As an administrator
+
+**Primary Actor:** An administrator.
+
+**Brief:** The user log is visible to the administrator.
+
+**Preconditions:**
+* The user log contains messages with varying tags.
```

Comment:
I mostly agree with you @guitarman0831 but I think that should be a separate use-case.

If no messages with tags exist, the assertion at the end that only messages with the users permission is unable to be verified.

I think this precondition should stay here and another case created for "when no messages exist, no messages are shown"

Thoughts?